rules/0095-sshd_rules.xml

```diff
@@ -144,7 +144,7 @@
     <group>invalid_login,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
   </rule>
 
-  <rule id="5720" level="10" frequency="8">
+  <rule id="5720" level="10" frequency="8" timeframe="300" ignore="120">
```

Comment:
Lots of rules about multiple authentication failures have the same issue: 30310, 81904, 9750, 85502, 80506, 81710, 40112, 40111, 85006, 85760

In addition, I think rules related to multiple authentication failures (probably a brute-force attack) should not be ignored.